EcalTiming/python/ecalLocalRecoSequenceAlCaStream_cff.py

```diff
@@ -0,0 +1,11 @@
+from RecoLocalCalo.Configuration.ecalLocalRecoSequence_cff import *
```

Comment:
Can we fix the implementation in the PhiSym package instead of duplicating things here?